dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/FileUtils.java

```diff
@@ -85,26 +85,36 @@ public static String getUploadFilename(String tenantCode, String filename) {
         return fileName;
     }
 
+    public static String getTenantDir(String tenant) {
+        return DATA_BASEDIR + "/exec/process/" + tenant;
```

Comment:
```suggestion
        return DATA_BASEDIR + EXEC_PROCESS + tenant;
```

Reply:
I remove this method, since we don't need this method.